### ISSUE TITLE
Who is re-tweeting?

### ISSUE THREAD

**Issue by swood-ecology** [Science-for-Nature-and-People/soc-twitter#24]
Create re-tweet network diagram of: 

1. For most 5 popular tweets within 2019 SHP conference, who tweeted it and who retweeted? In other words, create tweet/retweet network diagram of the 5 most re-tweeted tweets.
2. For 10 most popular overall tweets, create tweet/retweet network.
3. Create retweet network of users across all tweets.
4. Create retweet network of users across all tweets within each search term.

**Comment by meilinshi** [comment]
Preliminary results of 5 most retweeted tweets in SHS 2019
![image](https://user-images.githubusercontent.com/28639035/63960726-027f6100-ca44-11e9-90ef-3cd13c54db3b.png)
![image](https://user-images.githubusercontent.com/28639035/63960662-df54b180-ca43-11e9-86c8-a18e24ef364b.png)
- A couple of problems including labels, repeated nodes, etc.


![image](https://user-images.githubusercontent.com/28639035/63960675-e54a9280-ca43-11e9-8ebf-742b191541c5.png)

**Comment by swood-ecology** [comment]
I love this approach. The middle figure stands out because it has the labels. In addition to exploring the aspects in the original issue request, it would be cool to weight nodes by number of tweets and links by number of re-tweets between those two nodes.

**Comment by meilinshi** [comment]
I've changed the network diagram to interactive version to better show the names. Now nodes are weighted by number of tweets, links weighted by number of retweet between nodes. 
Data I'm using are all retweets from SHS 2019 conference
![Screen Shot 2019-09-04 at 5 02 59 PM](https://user-images.githubusercontent.com/28639035/64302023-4daedd80-cf36-11e9-9c83-6f25d51347de.png)

**Comment by swood-ecology** [comment]
This is fantastic! We'll have to figure out how to make the static figure look right, too, for putting in a journal article.

**Comment by meilinshi** [comment]
I didn't find better ways other than increasing fig size for the static figure. 
![000031](https://user-images.githubusercontent.com/28639035/64360382-39fb8980-cfbf-11e9-90f8-eb9ff8b0bdfe.png)

**Comment by swood-ecology** [comment]
Yeah we'll have to figure out the best way to do a static figure. What if you limited to the amount of RTs? Like, removed all tweet events that were not re-tweeted at least 5 times?

**Comment by meilinshi** [comment]
Updated with retweet_count > 5:
![image](https://user-images.githubusercontent.com/28639035/64730187-9402c000-d493-11e9-9072-e2d9d2ca84b8.png)
